docs/sdk/src/guides/your_first_node.rs

```diff
@@ -1 +1,312 @@
 //! # Your first Node
+//!
+//! In this guide, you will learn how to run a runtime, such as the one created in
+//! [`your_first_runtime`], in a node. Within the context of this guide, we will focus on running
+//! the runtime with an [`omni-node`]. Please first read this page to learn about the OmniNode, and
+//! other options when it comes to running a node.
+//!
+//! [`your_first_runtime`] is a runtime with no consensus related code, and therefore can only be
+//! executed with a node that also expects no consensus ([`sc_consensus_manual_seal`]).
+//! `polkadot-omni-node`'s [`--dev-block-time`] precisely does this.
+//!
+//! > All of the following steps are coded as unit tests of this module. Please see `Source` of the
+//! > page for more information.
+//!
+//! ## Running The Omni Node
+//!
+//! ### Installs
+//!
+//! The `polkadot-omni-node` can either be downloaded from the latest [Release](https://github.com/paritytech/polkadot-sdk/releases/) of `polkadot-sdk`,
+//! or installed using `cargo`:
+//!
+//! ```text
+//! cargo install polkadot-omni-node
+//! ```
+//!
+//! Next, we need to install the [`chain-spec-builder`]. This is the tool that helps us interact
+//! with the genesis related APIs of the runtime, as described in
+//! [`crate::guides::your_first_runtime#genesis-configuration`].
+//!
+//! ```text
+//! cargo install staging-chain-spec-builder
+//! ```
+//!
+//! ### Building Runtime
+//!
+//! Next, we need to build the corresponding runtime that we wish to interact with.
+//!
+//! ```text
+//! cargo build --release -p path-to-runtime
+//! ```
+//! Equivalent code in tests:
+#![doc = docify::embed!("./src/guides/your_first_node.rs", build_runtime)]
+//!
+//! This creates the wasm file under `./target/wbuild/release` directory.
+//!
+//! ### Building Chain Spec
+//!
+//! Next, we can generate the corresponding chain-spec file. For this example, we will use the
+//! `development` (`sp_genesis_config::DEVELOPMENT`) preset.
+//!
+//! Note that we intend to run this chain-spec with `polkadot-omni-node`, which is tailored for
+//! running parachains. This requires the chain-spec to always contain the `--para-id` and a
+//! `--relay-chain` fields, which are provided below.
+//!
+//! ```text
+//! chain-spec-builder \
+//! 	-c <path-to-output> \
+//! 	create \
+//! 	--para-id 42 \
+//! 	--relay-chain dontcare \
+//! 	--runtime polkadot_sdk_docs_first_runtime.wasm \
```

Comment:
So this is also because of the version discrepancy; my doc is explaining the master, and I think that it should stay like this for the sake of being "correct at the latest version". 

Ideally we would have versioning for the `polkadot-sdk-docs` crate as well, but for now we don't and in this scenario I suggest it should always follow the master branch. 

We used to have versioning in 

https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/index.html#

in the past, I will make an issue to explore reviving it.

Reply:
I thought `chain-spec-builder` is released from a different repository but it is part of `polkadot-sdk`. I think following the master branch is good enough for now as long as polkadot-sdk-docs is released at the same time with the referenced tooling (probably this is already the case). I think you can disregard my remarks.

I am uncertain if versioning of `polkadot-sdk-docs` will surely map to the right versions of the `polkadot-sdk` public APIs behaviour/business logic (even if we release them in lock step). I am thinking that for some stable releases of `polkadot-sdk` we could miss to update accordingly the `polkadot-sdk-docs`, which will follow at a later point even if that's not intended, for various reasons (mainly due to not having the docs populated "by design" like public API level docs). Reviving the versioning can be a good thing even for an imperfect process, but I wouldn't make it so (thinking of investing that effort in something more important) unless we find a certain use case that the current practice misses.